src/core/layout/scrollable-list/scrollable-list.js

```diff
@@ -101,9 +103,9 @@ const setupEvents = (scene, panel) => {
 const getPanelItems = panel => panel.getByName("grid", true).getElement("items");
 
 export class ScrollableList extends Phaser.GameObjects.Container {
-    constructor(scene) {
+    constructor(scene, context) {
```

Comment:
guessing `context` here is a string like "shop" or "manage"

the word `context` might get confused with something like `screen.context`